src/app/components/LiveLabel/index.tsx

```diff
@@ -34,14 +54,14 @@ const LiveLabel = ({
         dir={dir}
         {...(ariaHidden && { 'aria-hidden': 'true' })}
       >
-        {`${liveText} `}
+        {`${liveLabel} `}
       </span>
-      {offScreenText && (
+
+      {screenReaderText && (
         <VisuallyHiddenText lang={lang}>
-          {`${offScreenText}, `}
+          {`${screenReaderText}`}
```

Comment:
Can we simplify this?
```suggestion
          {screenReaderText}
```